bin/sl-deploy.js

```diff
@@ -56,16 +59,19 @@ while ((option = parser.getopt()) !== undefined) {
       local = true;
       break;
     default:
-      console.error('Invalid usage (near option \'%s\'), try `%s --help`.',
-        option.optopt,
-        $0);
+      console.error(GLB.t('msgInvalidUsage1', {
+        phOption: option.optopt,
```

Comment:
Why is there a leading `ph`? Why is there a leading `msg` in ever message? It just adds noise and boilerplate. Also, `1` and `2` as suffixes to distinguish messages is meaninless, and makes code unreadable.

Why not: 
```
glb.error('msgInvalidUsage1', {optopt: option.optopt, help: $0 + ' --help'});
```

Or even better:

glb.error('Invalid usage (near option \'`{opt}\'), try ``{$0}` --help`.', {
  opt: option.optopt,
  $0: $0}
  ); 
```